transformer/src/transform/dataframe.py

```diff
@@ -175,17 +172,28 @@ def merge_by_attributes(
                             )
                         merged_data[attribute.path] = input_group.static_inputs[0]
                 elif input_group.merging_script:
-                    # TODO I don't think the order of pyrog inputs is preserved here
-                    merged_data[attribute.path] = input_group.merging_script.apply(
-                        cur_attr_columns, input_group.static_inputs, attribute.path, primary_key,
+                    # TODO I don't think the order of pyrog inputs is preserved for merging scripts
+                    # If there is a merging for columns of length > 1, we want to merge
+                    # corresponding items together (first with first, etc.)
```

Comment:
I agree with the implementation but I think the doc could be improved, here's my suggestion:
```
When trying to merge several columns that each contain multiple values (represented as a list), we want to merge corresponding items together (first with first, etc.).
col1: [1, 2, 3]
       |  |  |
col2: [4, 5, 6]
       |  |  |
col3: [7, 8, 9]
```

Reply:
Thanks, I'll add that!

Reply:
not updated

Reply:
It wasn't pushed 😅